neo_force_scheme/neo_force_scheme.py

```diff
@@ -164,10 +164,20 @@ def _fit(self, X,
 
 
     def _transform(self, X, *, index, total, inplace, n_dimension: Optional[int] = 2, force_projection_dimensions=None,
-                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0)):
+                   original_z_axis=None, z_axis_moving_range: Optional[Tuple[float, float]] = (0, 0),
+                   confidence_interval: Optional[float] = 1):
         # iterate until max_it or if the error does not change more than the tolerance
         error = math.inf
 
+        z_score = 1
+        if confidence_interval < 1.0:
+            range_strict_limitation = False
+            get_z_score(confidence_interval)
```

Comment:
is this z_score ever used? I see its not attached to any variable

Reply:
Sorry for the confusion, it should be z_score =  get_z_score(confidence_interval), and then be passed to iteration()